deployment/frontend/src/components/dashboard/topics/TopicCard.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import SearchHeader from '../_shared/SearchHeader'
+import RowProfile from '../_shared/RowProfile';
+import Row from '../_shared/Row';
+import { PencilSquareIcon, TrashIcon } from '@heroicons/react/24/outline'
+import type { IRowProfile } from '../_shared/RowProfile';
+
+const teams = [
+  {
+    title: "Land and Carbon",
+    image: "/images/placeholders/topics/topic7.png",
+    num_datasets: 32,
+    description: '22 Subtopics',
+    subtopic: [
+      {
+        title: "Land and Carbon",
+        image: "/images/placeholders/topics/topic7.png",
+      },
+      {
+        title: "International Offices",
+        image: "/images/placeholders/topics/topic8.png",
+      },
+      {
+        title: "Global Offices",
+        image: "/images/placeholders/topics/topic1.png",
+      },
+      {
+        title: "Land and Carbon",
+        image: "/images/placeholders/topics/topic2.png",
+      },
+    ]
+  },
+  {
+    title: "International Offices",
+    image: "/images/placeholders/topics/topic8.png",
+    num_datasets: 32,
+    description: '22 Subtopics'
+  },
+  {
+    title: "Global Offices",
+    image: "/images/placeholders/topics/topic1.png",
+    description: '22 Subtopics'
+  },
+  {
+    title: "Land and Carbon",
+    image: "/images/placeholders/topics/topic2.png",
+    num_datasets: 32,
+    description: '22 Subtopics'
+  },
+  {
+    title: "International Offices",
+    image: "/images/placeholders/topics/topic3.png",
+    num_datasets: 32,
+    description: '22 Subtopics'
+  },
+  {
+    title: "Global Offices",
+    image: "/images/placeholders/topics/topic5.png",
+    description: '22 Subtopics'
+  }
+
+];
+
+function TeamProfile({ team }: { team: IRowProfile }) {
+
+  return (
+    <div className='flex py-3 pl-4 sm:pl-8'>
```

Comment:
I think this vertical padding should be py-5(in figma its 20px roughly which is closer to py-5) also the image inside should have rounded corners